exercises/en/test_03_11.py

```diff
@@ -0,0 +1,14 @@
+def test():
+    # Here we can either check objects created in the solution code, or the
+    # string value of the solution, available as __solution__. A helper for
+    # printing formatted messages is available as __msg__. See the testTemplate
+    # in the meta.json for details.
+
+    # If an assertion fails, the message will be displayed
+    assert 'random_state=33', "Make sure you are settting the random state to 33 when splittting the data."
+    assert 'test_size=0.2' in __solution__, "Make sure you are doing a 20-80 split."
```

Comment:
I wouldn't test this one or I would make it and "or" condition and say train_size=0.8.